source/common/access_log/access_log_formatter.cc

```diff
@@ -136,10 +142,16 @@ std::vector<FormatterPtr> AccessLogFormatParser::parse(const std::string& format
 
         formatters.emplace_back(
             FormatterPtr(new ResponseHeaderFormatter(main_header, alternative_header, max_length)));
+      } else if (token.find("START_TIME") == 0) {
+        const size_t start = 11;
```

Comment:
Is this `"START_TIME".size()`? Maybe just use some static constant to that effect?

Reply:
yes, and agreed. Should I do the same for lines 130 and 146, or leave those untouched as out-of-scope for this issue/PR?

Reply:
Sure, that would be a nice cleanup.